src/vm/perfmap.cpp

```diff
@@ -12,6 +12,12 @@
 #include "perfinfo.h"
 #include "pal.h"
 
+#ifdef _TARGET_64BIT_
```

Comment:
This works, but it will make them file less readable. I would keep the crossgen addresses/offsets padded to 32-bit. Something like:
 
```
#ifdef CROSSGEN_COMPILE
// The code addresses are actually native image offsets during crossgen. Print them as 32-bit numbers for consistent output when cross-targeting and to make the output more compact.
#define FMT_TARGET_ADDR "%0x"
#else
#define FMT_TARGET_ADDR "%p"
#endif
```